File: documentation/user/en/solve/examples/render-category-menu/sub-categories-listing.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		EntityLocaleEquals(CultureInfo.GetCultureInfo("en")),
        		HierarchyWithin(
        			"categories",
        			AttributeEquals("url", "/en/accessories")
        		)
        	),
        	Require(
        		HierarchyOfReference(
        			"categories",
        			RemoveEmpty,
        			Children(
        				"subcategories",
        				EntityFetch(
        					AttributeContent("code")
        				),
        				StopAt(
        					Distance(1)
        				)
        			)
        		),
        		Page(1, 0)
        	)
        )
	)
);

File: documentation/user/en/solve/examples/render-category-menu/sub-categories-listing.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					entityLocaleEquals(Locale.forLanguageTag("en")),
					hierarchyWithin(
						"categories",
						attributeEquals("url", "/en/accessories")
					)
				),
				require(
					hierarchyOfReference(
						"categories",
						children(
							"subcategories",
							entityFetch(
								attributeContent("code")
							),
							stopAt(
								distance(1)
							)
						)
					),
					page(1, 0)
				)
			)
		);
	}
);
